### ISSUE TITLE
What means `[main/ERROR] [mixin]: Classloader restrictions [PACKAGE_TRANSFORMER_EXCLUSION]`?

### ISSUE THREAD

**Issue by Krutoy242** [CleanroomMC/ModularUI#37]
Loading `E2E-E` with ModularUI print this errors in debug.log file:

```less
[main/ERROR] [mixin]: Classloader restrictions [PACKAGE_TRANSFORMER_EXCLUSION] encountered loading mixin.modularui.json:SlotMixin from mod unknown-owner, name: com.cleanroommc.modularui.core.mixin.SlotMixin
[main/ERROR] [mixin]: Classloader restrictions [PACKAGE_TRANSFORMER_EXCLUSION] encountered loading mixin.modularui.json:GuiContainerAccessor from mod unknown-owner, name: com.cleanroommc.modularui.core.mixin.GuiContainerAccessor
[main/ERROR] [mixin]: Classloader restrictions [PACKAGE_TRANSFORMER_EXCLUSION] encountered loading mixin.modularui.json:GuiContainerMixin from mod unknown-owner, name: com.cleanroommc.modularui.core.mixin.GuiContainerMixin
[main/ERROR] [mixin]: Classloader restrictions [PACKAGE_TRANSFORMER_EXCLUSION] encountered loading mixin.modularui.json:KeyBindAccess from mod unknown-owner, name: com.cleanroommc.modularui.core.mixin.KeyBindAccess
[main/ERROR] [mixin]: Classloader restrictions [PACKAGE_TRANSFORMER_EXCLUSION] encountered loading mixin.modularui.json:MinecraftMixin from mod unknown-owner, name: com.cleanroommc.modularui.core.mixin.MinecraftMixin
[main/ERROR] [mixin]: Classloader restrictions [PACKAGE_TRANSFORMER_EXCLUSION] encountered loading mixin.modularui.json:SlotAccessorClient from mod unknown-owner, name: com.cleanroommc.modularui.core.mixin.SlotAccessorClient
```

What does them means? Is that means some of GUIs or functionality of BogoSorter wouldnt work?

**Comment by brachy84** [comment]
I dont know

**Comment by Krutoy242** [comment]
But isn't this is mixins from modularUI? 😅

**Comment by brachy84** [comment]
That doesnt mean i know what that is. But now i know what it is. Ill fix it in 2.3.1
